src/aws/lambda.tf

```diff
@@ -58,6 +58,9 @@ resource "aws_lambda_function" "marsha_encode_lambda" {
   filename         = "dist/marsha_encode.zip"
   source_code_hash = "${base64sha256(file("dist/marsha_encode.zip"))}"
   role             = "${aws_iam_role.lambda_invocation_role.arn}"
+  memory_size      = "1536"
+  timeout          = "30"
```

Comment:
What is the typical processing time?

Reply:
We don't know, we should monitor this to know if it's enough or not. In my test it was between 10 and 15 seconds.

Reply:
So maybe increase this value to at least x3 the expected execution time.

Reply:
:ok_hand: